docs/_interface/sync-modes.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Sync-modes
+sort-key: L
+---
+
+Syncing is the process by which Geth catches up to the latest Ethereum block and current global state. 
+There are several ways to sync a Geth node that differ in their speed, storage requirements and trust 
+assumptions. This page outlines three sync configurations for full nodes and one for light nodes.
+
+## Full nodes
+
+There are two types of full node that use different mechanisms to sync up to the head of the chain:
+
+### Snap (default)
+
+A snap sync'd node holds the most recent 128 blocks in memory, so transactions in that range are always 
+accessible. However, snap-sync only starts processing from a relatively recent block (as opposed to genesis 
+for a full node). Between the initial sync block and the 128 most recent blocks, the node stores occasional 
```

Comment:
The periodically flush is actually the in-memory state pruning mechanism. It's not relevant with snap sync at all. In another word, no matter it's full sync'd, snap sync'd or even legacy fast sync'd, they will also periodically flush state into disk and prune all others unless `gcmode = archive`.

I will vote to explain a bit more snap-sync specified maybe.

Reply:
for example, we can explain the workflow of the snap sync, it can be divided into different stages
- backward header sync(with some security consideration, etc)
- backfill corresponding block bodies/receipts in between
- at the same time, retrieve the state data

Reply:
In state sync, it can also be divided to two stages:
- sync stage: retrieve raw state data(accounts, storage slots) with range proof attached
- healing stage: fulfill the missing trie nodes because of the syncing target moving